portfolio/src/main/webapp/projects/musicplayer/player.html

```diff
@@ -0,0 +1,59 @@
+<html>
+<header>
+   <title>Music Player</title>
+   <link rel="stylesheet" type="text/css" href="ipod.css"/>
+   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</header>
+<body>
+   <div id="screen">
+      <div class="head-bar">
+         <a class= "back-button" href="songs.html"><i class="material-icons">arrow_back</i></a>
+      </div>
+      <div class="player-album-art">
+        <img src="images/cbook.jpg" alt="Coloring Book Album Cover" height=240 widht=240/>
+      </div>
+
+      <div class="player-song-info">
+        <p id="player-song-name">Mixtape</p>
+        <p><em>Coloring Book</em> - Chance the Rapper</p>
+      </div>
+
+      <div><input type="range" id="player-time" value="0" min="0" max="180" onchange="timeElapsed()"> </div> <!used onchange to change time-elapsed with value of player-time>
+      <span id="time-elapsed">0:00</span>  
+      <span id="time-total">3:00</span> 
+
+
+      <div class="player-controls">
+        <button onclick="prevSong()"><i class="fa fa-backward fa-lg"></i></button>
+        <button id="play-button" onclick="switchPlay()"><i id="play-icon" class="fa fa-play fa-lg"></i></button>
```

Comment:
<i> is for italics/emphasis (though it can be styled any way through css). For icons handled as the background of an element, it's probably better to use span or div.